src/Microsoft.Dnx.Runtime/ApplicationHostContext.cs

```diff
@@ -52,6 +56,11 @@ public static IList<LibraryDescription> GetRuntimeLibraries(ApplicationHostConte
                 }
             }
 
+            if (!string.IsNullOrEmpty(result.MinRequiredRuntimeVersion))
```

Comment:
This confused me at first because I thought it was always being set to the minimum version (so I was like "but what if it's below the current runtime version?"). Can you change the below code to just extract the version and put the comparison here instead?